src/UI/WB.UI.Frontend/src/webinterview/components/Navbar.vue

```diff
@@ -229,14 +229,7 @@ export default {
             $('.top-menu').toggleClass('top-animate')
             $('.mid-menu').toggleClass('mid-animate')
             $('.bottom-menu').toggleClass('bottom-animate')
-            if ($(window).width() < 880) {

```

Comment:
please check code  on 217 line
and also check compatibility.js file. on the same code

Reply:
everything was removed